components/vocabularies/orphanet/ui/src/main/resources/PhenoTips/OrphanetInfoService.xml

```diff
@@ -0,0 +1,75 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I'd rather not add new files for every vocabulary, but start using the REST services, if possible.

Same comment for the next file.